### DISCUSSION TITLE
A question about `bb_fix`

### DISCUSSION THREAD

**Discussion by yilewang** [McIntosh-Lab/tvb-ukbb#180] [General] [answered]
Hi! Happy new year to everyone~

I just have a general question. In the pipeline, when we use `fix` to do the preprocessing, the only training file provided inside the pipeline is from `ADNI3` dataset. I am wondering if I use `ADNI3` training data to preprocess the normal controls, will it generate bias and inaccuracy in my preprocessing? Do you have any suggestions for any other training files for normal controls, or even better, the `fix` training file for aging normal controls? 

Best Regards, 

Yile

**Comment by shen4brains** [comment, marked as answer]
Hi @yilewang, happy new year!

Ideally, for `fix`, you should use your own data to generate a training set. If your data are similar enough to data from available training sets, including the few that we provide, then it could perform accurately. There are other training sets available as a part of the `fix` distribution, which you can find more on [here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX/UserGuide#Training_datasets). It's best to check `fix` performance on a few subjects to see how accurately it is performing and whether you need to customize your training set. We've written a bit on this topic in our FAQ section [here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX/UserGuide#Training_datasets).
